metronome/hotstuff/consensus/src/io/iohk/metronome/hotstuff/consensus/ViewNumber.scala

```diff
@@ -10,4 +11,7 @@ object ViewNumber extends Tagger[Long] {
 
   implicit val ord: Ordering[ViewNumber] =
     Ordering.by(identity[Long])
+
+  implicit val order: Order[ViewNumber] =
+    Order.fromOrdering(ord)
```

Comment:
Just curious, what does `Order` gives us, that `Ordering` does not?

Reply:
Had to comment it to to see what fails...

It's [maximumBy](https://typelevel.org/cats/api/cats/NonEmptyReducible.html#maximumBy[A,B](fa:F[A])(f:A=%3EB)(implicitevidence$3:cats.Order[B]):A) which I used [here](https://github.com/input-output-hk/metronome/pull/36/files#diff-e5099455fe288ae75f26d96087e70bb166397f06b80c0ba11201605a3ed9cbf3R138-R139) on `NonEmptySeq` instances.